candle-examples/examples/trocr/readme.md

```diff
@@ -0,0 +1,16 @@
+# candle-trocr
+
+`TrOCR` is a transformer OCR Model. In this example it is used to
+transcribe image text. See the associated [model
+card](https://huggingface.co/microsoft/trocr-base-printed) for details on
+the model itself.
+
+## Running an example
+
+```bash
+cargo run --example trocr --release --  --which base --cpu --image assets/trocr.png  --tokenizer tokenizer.json
```

Comment:
Could you remove the tokenizer argument here and instead retrieve it by default from the hub if that's not already the case. Also better to reduce the number of arguments and use the default values where possible.

Reply:
The tokenizer is missing a `tokenizer.json` on hub so i had to manually save to file. Is there another of loading the tokenizer without that file?

Reply:
You can just create a `candle-trocr-tokenizer` model on the hf hub, upload the tokenizer config there so that future users don't have to do anything to get this file.

Reply:
Done!